doc/how_to_docs.rst

```diff
@@ -58,3 +65,8 @@ Then the following commands allow you to create the documentation locally:
 
 This will create html files in `pypsa-earth/doc/_build/html`.
 VScode provides a so called Liveserver extension such that the html file can be opened locally on your computer.
+
+.. note::
+    To build the documentation, Windows users might need to replace the last command by:
```

Comment:
I think between line 69 and 70 you need a empty line

Reply:
thank you, done.